Source/WebKit/WebProcess/GPU/webrtc/LibWebRTCCodecs.cpp

```diff
@@ -124,10 +126,10 @@ static int32_t registerDecodeCompleteCallback(webrtc::WebKitVideoDecoder::Value
 
 static webrtc::WebKitVideoEncoder createVideoEncoder(const webrtc::SdpVideoFormat& format)
 {
-    if (format.name == "H264")
+    if (format.name == "H264" || format.name == "h264")
```

Comment:
Is there a reason to use `equalIgnoringASCIICase` in `createVideoDecoder` and test both variants explicitly here?

Reply:
It is a minor optimisation, we do not need to copy the string in WTFString to use equalIgnoringASCIICase. We cannot do this trick in the other cases because of 'VP9' and 'AV1' 4 variants.